supervisor/hardware/manager.py

```diff
@@ -101,10 +101,22 @@ def _import_devices(self) -> None:
         self._devices.clear()
 
         # Exctract all devices
-        for device in self._udev.list_devices():
+        for device in self._udev.list_devices():            
```

Comment:
Drop this unrelated change:

```suggestion
        for device in self._udev.list_devices():
```

Reply:
Thanks @agners 

I done your changes and verified - does what it says on the tin.

```
2024-05-22 14:43:04.000 INFO (MainThread) [__main__] Setting up Supervisor
2024-05-22 14:43:04.235 INFO (MainThread) [supervisor.api] Starting API on 172.30.32.2
2024-05-22 14:43:04.472 WARNING (MainThread) [supervisor.hardware.manager] Ignoring udev device due to error: 'utf-8' codec can't decode byte 0x9f in position 7: invalid start byte
2024-05-22 14:43:04.517 INFO (MainThread) [supervisor.dbus.manager] Connected to system D-Bus.
2024-05-22 14:43:04.519 INFO (MainThread) [supervisor.dbus.agent] Load dbus interface io.hass.os
```